cli/Cargo.toml

```diff
@@ -48,7 +50,6 @@ tar = "0.4.37"
 thiserror = "1.0.30"
 ureq = { version = "2.3.1", features = ["json"] }
 nix = "0.23.0"
-log = "0.4.14"
```

Comment:
To be honest, I've been looking forward to removing `log` for a long time.
So, is it possible to remove env_logger?

Reply:
Consider remove it in another PR